### DISCUSSION TITLE
重启后打开小浮窗时 自动恢复到上次关闭小浮窗时的位置

### DISCUSSION THREAD

**Discussion by Yersmagit** [Class-Widgets/Class-Widgets#302] [Feature]
### 检查清单

- [X] 我已更新到最新版 ![最新的版本号](https://img.shields.io/github/v/release/RinLit-233-shiroko/Class-Widgets?include_releases&style=flat-square&label=) ，并看过[最新提交](https://github.com/RinLit-233-shiroko/Class-Widgets/commits/main/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/RinLit-233-shiroko/Class-Widgets/issues?q=label%3ABug) 中检索，确认这一 Bug 未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 **软件本来** 的开发目标，而非 **添加与课表及教学辅助无关的内容** 。
- [X] 我已知晓由于开发者时间精力有限，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

每次打开软件第一次开小浮窗都要重新调整位置，略有稍许不便。

### 想要实现或优化的功能

重启软件后，打开小浮窗，可以自动恢复到上次关闭小浮窗时的位置。
同时建议在设置中添加一个“重置小浮窗默认位置”的选项，以防止出现意外的问题。

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

**Comment by RinLit-233-shiroko** [comment]
好

**Comment by IsHPDuwu** [comment]
https://github.com/Class-Widgets/Class-Widgets/pull/444
